WordPress/src/main/java/org/wordpress/android/ui/publicize/PublicizeActions.java

```diff
@@ -114,7 +123,17 @@ private static void connectStepOne(final long siteId, final String serviceId, fi
         RestRequest.Listener listener = new RestRequest.Listener() {
             @Override
             public void onResponse(JSONObject jsonObject) {
-                if (shouldShowChooserDialog(siteId, serviceId, jsonObject)) {
+                final boolean showChooserDialog;
+                try {
+                    showChooserDialog = shouldShowChooserDialog(siteId, serviceId, jsonObject);
+                } catch (PublicizeConnectionValidationException e) {
+                    final ActionCompleted event =
```

Comment:
Nice work here generally :)
Just wondering, should we maybe print the error in the log so that we can track in while debugging?

Reply:
Thanks 🙂 

Do you mean the error encapsulated by `PublicizeConnectionValidationException`? 🤔  Hmm, for `PublicizeConnectionValidationException`, the error is always an `int` (`mReasonResId`) which, for now, points to `R.string.sharing_facebook_account_must_have_pages`. To log that, we'd need to have a `Context` to get the full string. 

Do you think it's okay to add something like this after line 216 instead?

```java
AppLog.i(AppLog.T.SHARING, "Could not connect the Facebook account because there is no Page selected.");
```

The string will be hardcoded in the code but I think it's fine.

Reply:
Yeah, if it's ok with you something like that would be perfect! Thanks!